### PR TITLE
DBZ-1356 Metrics for schema history

### PR DESCRIPTION
**Pull request by jpechane**
https://issues.jboss.org/browse/DBZ-1356

### CODE REVIEW

File: debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/Module.java [hunk 1]
```diff
@@ -25,4 +25,8 @@ public static String version() {
     public static String name() {
         return "oracle";
     }
+
+    public static String connectorName() {
```
Comment (gunnarmorling) [inline debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/Module.java]: @jpechane so what's the different usages of `name()` vs. `connectorName()`. Wondering whether we can consolidate those two actually?

Comment (jpechane) [reply debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/Module.java]: @gunnarmorling One is used in JMX metrics and the other in sourceinfo. Unfrotunately some connectors have different vale for these two so for backward compatibility I'd keep them separated.

Comment (gunnarmorling) [reply debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/Module.java]: It seems though `name()` could be derived from `connectorName()` by lower-casing and removing the "_"? I.e. `name()` could look like so in the base class:

```
public static String name() {
        return connectorName().toLowerCase().replaceAll("_", "");
}
```

WDYT?

And the two methods should have JavaDoc on their purpose in the base class, too.

Comment (jpechane) [reply debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/Module.java]: @gunnarmorling I thought about it too but check postgres module file

Comment (jpechane) [reply debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/Module.java]: @gunnarmorling And do not forget that static method calls have early not late binding so inheritance does not work.

Comment (jpechane) [reply debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/Module.java]: @gunnarmorling Thinking about it more, postgresql metrics are not exposed yet. The JMX metrics names are lower-cased anyway so the only porblem left is SQLServer that has metrics exposed with `_`.
The other impact is just the logging context name in log files which I believe we cann afford to change

Comment (gunnarmorling) [reply debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/Module.java]: On SQL Server, see above: couldn't we simply exclude the "_"? It seems a reasonably generic thing to do.

**Pull request merged**